docs/ibc/Slashing.md

```diff
@@ -0,0 +1,7 @@
+# Slashing Evidence Handling
```

Comment:
IMO slashing may have to be it's own section?

There is a lot to discuss with regards to this, and different types of slashing that could be implemented in the future.

Reply:
I gave it it's own page. We can move it to a section later.

This part is on how to prove it cross-chain. Actually what we do when proven needs more as well for sure.